spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2671,6 +2678,13 @@ def setup_context_menu(self):
             self, _("Zoom reset"), shortcut=QKeySequence("Ctrl+0"),
             triggered=self.zoom_reset.emit)
 
+        # Docstring
+        writer = self.writer_docstring
+        self.docstring_action = create_action(
+            self, _("Generate function docstring"),

```

Comment:
Sorry for changing my mind on this, but ``Generate docstring`` would be just as clear and more general (since it already generates method docstrings, and at some point could be extended to classes and modules.